testing-guidance/workflow-tests/remote-tools-connection-gem/remote-tools-connection-gem-workflow-tests.md

```diff
@@ -10,17 +10,34 @@ This gem enables connection to the Editor to other remote tools. Currently the g
 
 ## Workflows
 
-| Workflow                                                                  | Steps                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | What to look for                                                                           |
```

Comment:
Multiple lines in the "Step" column are missing punctuation at the end of each step; some have it, some don't. 
These should be changed to remain consistent.